daemon/container_operations_windows.go

```diff
@@ -35,6 +40,58 @@ func detachMounted(path string) error {
 	return nil
 }
 
+func (daemon *Daemon) setupSecretDir(c *container.Container) (setupErr error) {
+	if len(c.SecretReferences) == 0 {
+		return nil
+	}
+
+	localMountPath := c.SecretMountPath()
+	logrus.Debugf("secrets: setting up secret dir: %s", localMountPath)
+
+	defer func() {
+		if setupErr != nil {
+			if err := os.RemoveAll(localMountPath); err != nil {
+				if !os.IsNotExist(err) {
+					logrus.Errorf("error cleaning up secret mount: %s", err)
+				}
+			}
+		}
+	}()
+
+	// create local secret root
+	if err := system.MkdirAllWithACL(localMountPath, 0); err != nil {
+		return errors.Wrap(err, "error creating secret local directory")
+	}
+
+	for _, s := range c.SecretReferences {
+		if c.SecretStore == nil {
+			return fmt.Errorf("secret store is not initialized")
+		}
+
+		// TODO (ehazlett): use type switch when more are supported
```

Comment:
To clarify; yes, we use TODO's in the code; usually not for "unfinished" code, but more as reminders that (e.g.) something needs to be removed after release X, or after feature Y is implemented. The `TODO (name)` format also works in some editors/IDE's to show a todo list for one person specific :smile: https://confluence.jetbrains.com/display/PhpStorm/Working+with+todo+comments+and+the+todo+tool+window